app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -248,20 +248,25 @@ class LuciBuildService {
   ///
   /// Builds are queried based on the [RepositorySlug] and pull request number.
   Future<void> cancelBuilds(github.PullRequest pullRequest, String reason) async {
+    log.info(
+        'Attempting to cancel builds... for pullrequest ${pullRequest.base!.repo!.fullName}/${pullRequest.number}');
     final Map<String?, Build?> builds = await tryBuildsForPullRequest(pullRequest);
-    if (!builds.values.any((Build? build) {
-      return build!.status == Status.scheduled || build.status == Status.started;
-    })) {
-      return;
-    }
+
     final List<Request> requests = <Request>[];
     for (Build? build in builds.values) {
-      requests.add(
-        Request(
-          cancelBuild: CancelBuildRequest(id: build!.id, summaryMarkdown: reason),
-        ),
-      );
+      // Scheduled status includes scheduled and pending tasks.
+      if (build!.status == Status.scheduled || build.status == Status.started) {
```

Comment:
Should we also add pending here?

Reply:
The Scheduled status includes both.

Reply:
From this comment in the source: https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/model/luci/buildbucket.dart;l=862
I can certainly add it if that is what you mean.

Reply:
See my previous comment, it is not in the status enum.